packages/@okta/vuepress-site/code/dotnet/aspnet/index.md

```diff
@@ -19,6 +19,11 @@ meta:
 			<i class='icon code-dotnet-32'></i><span>ASP.NET</span>
 		</RouterLink>
 	</li>
+	<li>
+		<RouterLink to='/code/dotnet/blazor/'>
+			<i class='icon code-dotnet-32'></i><span>Blazor</span>
```

Comment:
@swiftone how do we add a new icon? Here is the new one for Blazor, but we will have others too like Xamarin
https://devblogs.microsoft.com/aspnet/wp-content/uploads/sites/16/2019/04/BrandBlazor_nohalo_1000x.png